app/forms/Validation.scala

```diff
@@ -19,7 +19,8 @@ package forms
 object Validation {
   final val EMAIL_REGEX =
     """^(?!\.)("([^"\r\\]|\\["\r\\])*"|([-a-zA-Z0-9!#$%&'*+/=?^_`{|}~]|(?<!\.)\.)*)(?<!\.)@[a-zA-Z0-9][\w\.-]*[a-zA-Z0-9]\.[a-zA-Z][a-zA-Z\.]*[a-zA-Z]$"""
-  final val GROUPID_REGEX = "^X[A-Z]PLR[0-9]{10}$"
+  final val GROUPID_REGEX                = "^X[A-Z]PLR[0-9]{10}$"
+  final val REGISTRATION_TELEPHONE_REGEX = """^[A-Z0-9 )/(\-*#+]+$"""
```

Comment:
```suggestion
  final val GROUPID_REGEX                = "^X[A-Z]PLR[0-9]{10}$"
  final val REGISTRATION_TELEPHONE_REGEX = "^[A-Z0-9 )/(\-*#+]+$"
```
For the sake of consistency, can you use `"` or `"""`, though I see that most patterns are using single quotes.

Reply:
Background: this is a problem regex as the `-` is confused with starting a range. I was on a call with @JamesMMiller and I didn't like the idea of making changes to the regex, so James suggested we use smart strings to bypass escaping and that worked. Single quotes don't work here, the compiler deems the `\-` invalid and the code wouldn't compile, feel free to have a go at it and let me know if you have any luck

Reply:
By the way, I liked the idea of `Commit suggestion`, you need to show me how you did it when we meet in the office :v: